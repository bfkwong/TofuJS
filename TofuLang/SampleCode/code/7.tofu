x = true; 

if (x) is true then {
  print(true);
}

if (x == true) is true then {
  print(true);
} 

x = 10; 
if (x == 0) is true then {
  print("0");
} or else {
  print("not 0");
}