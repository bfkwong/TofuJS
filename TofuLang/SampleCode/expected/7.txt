true
true
not 0
